WcaOnRails/app/views/users/edit.html.erb

```diff
@@ -172,6 +172,12 @@
       <%= f.input :region, disabled: !editable_fields.include?(:region) %>
     <% end %>
 
+    <div class="form-group">
+      <% @user.teams.each do |team| %>
+        <%= link_to team.name, edit_team_path(team), class: "btn btn-info #{"disabled" unless current_user.can_edit_team?(team)}" %>
```

Comment:
Maybe something like this:
![image](https://cloud.githubusercontent.com/assets/17034772/14587933/a08c7fbe-04bc-11e6-82a4-81c73cf6f464.png)